engine/runtime-integration-tests/src/test/scala/org/enso/interpreter/test/instrument/RuntimeErrorsTest.scala

```diff
@@ -2428,7 +2428,7 @@ class RuntimeErrorsTest
         contextId,
         mainResId,
         Api.ExpressionUpdate.Payload.Panic(
-          "Compile_Error",
+          "Compile_Error.Error",
```

Comment:
Why has this changed?

Reply:
I was about to ask @4e6 to give me a clue why this change is needed. I have no idea which part is responsible for providing this panic payload.